crates/bevy_render/src/mesh/allocator.rs

```diff
@@ -119,6 +119,21 @@ pub struct MeshAllocatorSettings {
     pub growth_factor: f64,
 }
 
+impl Default for MeshAllocatorSettings {
+    fn default() -> Self {
+        Self {
+            // 1MB
```

Comment:
This should be MiB since it's 1024 and not 1000 no?

Reply:
I don't like the use of MiB but I changed it anyway.